test/spec/modules/emx_digitalBidAdapter_spec.js

```diff
@@ -4,167 +4,343 @@ import * as utils from 'src/utils';
-            ]
-          }
-        }
-      })).to.equal(false);
-      expect(spec.isBidRequestValid({
-        bidder: 'emx_digital',
-        params: {
-          tagid: ''
+    describe('video request validity', function () {
+      let bid = {
+        'bidder': 'emx_digital',
+        'params': {
+          'tagid': '25251',
+          'video': {}
         },
-        mediaTypes: {
-          banner: {
-            sizes: [
-              [300, 250],
-              [300, 600]
-            ]
+        'mediaTypes': {
+          'video': {
+            'context': 'instream',
+            'playerSize': [640, 480]
           }
-        }
-      })).to.equal(false);
-      expect(spec.isBidRequestValid({
-        bidder: 'emx_digital',
-        params: {
-          tagid: '123'
         },
-        mediaTypes: {
-          banner: {
-            sizes: [
-            ]
+        'adUnitCode': 'adunit-code',
+        'sizes': [
+          [300, 250],
+          [300, 600]
+        ],
+        'bidId': '30b31c2501de1e',
+        'bidderRequestId': '22edbae3120bf6',
+        'auctionId': '1d1a01234a475'
+      };
+      let noInstreamBid = {
+        'bidder': 'emx_digital',
+        'params': {
+          'tagid': '25251',
+          'video': {
+            'protocols': [1, 7]
           }
-        }
-      })).to.equal(false);
-      expect(spec.isBidRequestValid({
-        bidder: 'emxdigital',
-        params: {
-          tagid: '123'
         },
-        mediaTypes: {
-          banner: {
-            sizes: [
-              [300, 250],
-              [300, 600]
-            ]
+        'mediaTypes': {
+          'video': {
+            'context': 'something_random'
           }
-        }
-      })).to.equal(false);
-      expect(spec.isBidRequestValid({
-        bidder: 'emx_digital',
-        params: {
-          tagid: '123'
         },
-        mediaTypes: {
-          banner: {
-            sizes: [
-              [300, 250],
-              [300, 600]
-            ]
+        'adUnitCode': 'adunit-code',
+        'sizes': [
+          [300, 250],
+          [300, 600]
+        ],
+        'bidId': '30b31c2501de1e',
+        'bidderRequestId': '22edbae3120bf6',
+        'auctionId': '1d1a01234a475'
+      };
+
+      it('should return true when required params found', function () {
+        expect(spec.isBidRequestValid(bid)).to.equal(true);
+        expect(spec.isBidRequestValid(noInstreamBid)).to.equal(false);
+      });
+
+      // it('bidRequest with missing instream video context', function() {
+      //   const testBid = Object.assign({}, bid);
+      //   delete testBid.mediaTypes.video.context;
+      //   expect(spec.isBidRequestValid(testBid)).to.equal(false);
+      // })
+
+      // it('bidRequest with missing video playerSize', function() {
+      //   const testBid = Object.assign({}, bid);
+      //   delete testBid.mediaTypes.video.playerSize;
+      //   expect(spec.isBidRequestValid(testBid)).to.equal(false);
+      // })
```

Comment:
Are these tests something you want to reactivate?  Or remove?

Reply:
Thanks, we've removed these lines in our latest commit.